<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdqPh1_7</class>
 <widget class="QMainWindow" name="AdqPh1_7">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>861</width>
      <height>581</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Adquisition</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_19">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>271</width>
        <height>231</height>
       </rect>
      </property>
      <property name="title">
       <string>Photo Camera 1</string>
      </property>
      <widget class="QLabel" name="AdqPhotoFrame1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>251</width>
         <height>201</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_20">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>310</y>
        <width>271</width>
        <height>231</height>
       </rect>
      </property>
      <property name="title">
       <string>Photo Camera 3</string>
      </property>
      <widget class="QLabel" name="AdqPhotoFrame3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>251</width>
         <height>201</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_21">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>310</y>
        <width>271</width>
        <height>231</height>
       </rect>
      </property>
      <property name="title">
       <string>Photo Camera 2</string>
      </property>
      <widget class="QLabel" name="AdqPhotoFrame2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>251</width>
         <height>201</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>271</width>
        <height>231</height>
       </rect>
      </property>
      <property name="title">
       <string>Camera 1</string>
      </property>
      <widget class="QLabel" name="AdqVideoFrame1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>251</width>
         <height>201</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="CalPBCaptureAd">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Photo Capture</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>10</y>
        <width>271</width>
        <height>231</height>
       </rect>
      </property>
      <property name="title">
       <string>Camera 3</string>
      </property>
      <widget class="QLabel" name="AdqVideoFrame3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>251</width>
         <height>201</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>10</y>
        <width>271</width>
        <height>231</height>
       </rect>
      </property>
      <property name="title">
       <string>Camera 2</string>
      </property>
      <widget class="QLabel" name="AdqVideoFrame2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>251</width>
         <height>201</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="AdqPh">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>260</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Get Photo</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="AdqPh_ma">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>260</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="AdqPh_me">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>260</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_secuencia">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>270</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Alignment Cam 1 - Cam 2</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>551</width>
        <height>271</height>
       </rect>
      </property>
      <property name="title">
       <string>Camera 1 and Camera 2 matches</string>
      </property>
      <widget class="QLabel" name="AlgMatch1_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>531</width>
         <height>241</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_22">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>310</y>
        <width>271</width>
        <height>231</height>
       </rect>
      </property>
      <property name="title">
       <string>Photo 2 Cropped</string>
      </property>
      <widget class="QLabel" name="AlgCrop2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>251</width>
         <height>201</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_23">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>271</width>
        <height>231</height>
       </rect>
      </property>
      <property name="title">
       <string>Photo 1 Cropped</string>
      </property>
      <widget class="QLabel" name="AlgCrop1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>251</width>
         <height>201</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>50</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maxVisibleItems">
       <number>2</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>45</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Descriptor:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="CLG1">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>100</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Calibrar</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_26">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>310</y>
        <width>271</width>
        <height>231</height>
       </rect>
      </property>
      <property name="title">
       <string>Fusion 1 Photo</string>
      </property>
      <widget class="QLabel" name="AlgFusion1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>251</width>
         <height>201</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="DCG1">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>170</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Recortar y Fusionar</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Alignment Cam 2 - Cam 3</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>45</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Descriptor:</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_25">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>310</y>
        <width>271</width>
        <height>231</height>
       </rect>
      </property>
      <property name="title">
       <string>Photo 3 Cropped</string>
      </property>
      <widget class="QLabel" name="AlgCrop3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>251</width>
         <height>201</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>551</width>
        <height>271</height>
       </rect>
      </property>
      <property name="title">
       <string>Camera 2 and Camera 3 matches</string>
      </property>
      <widget class="QLabel" name="AlgMatch2_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>531</width>
         <height>241</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="CLG2">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>100</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Calibrar</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>50</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_27">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>271</width>
        <height>231</height>
       </rect>
      </property>
      <property name="title">
       <string>Photo 2 Cropped</string>
      </property>
      <widget class="QLabel" name="AlgCropFusion1_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>251</width>
         <height>201</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_24">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>310</y>
        <width>271</width>
        <height>231</height>
       </rect>
      </property>
      <property name="title">
       <string>Fusion 2 Photo</string>
      </property>
      <widget class="QLabel" name="AlgCropFusion2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>251</width>
         <height>201</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="DCG2">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>170</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Alinear, Recortar y Fusionar</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>IMAGE FUSION TOOL</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
